http-client/Network/HTTP/Client/Manager.hs

```diff
@@ -286,6 +287,16 @@ noProxy = ProxyOverride $ const $ return $ \req -> req { proxy = Nothing }
 useProxy :: Proxy -> ProxyOverride
 useProxy p = ProxyOverride $ const $ return $ \req -> req { proxy = Just p }
 
+-- | Send secure requests to the proxy in plain text rather than using CONNECT,
+-- regardless of the value in the @Request@.
+--
+-- Since 0.7.2
```

Comment:
For new code, I'd prefer to use the new `@since` syntax (I realize the other code doesn't use that.

Can you also make a cabal file bump and ChangeLog entry about this?